src/core/exec-invoke.c

```diff
@@ -4562,8 +4566,9 @@ int exec_invoke(
          * disallocate the VT), to get rid of any prior uses of the device. Note that we do not keep any fd
          * open here, hence some of the settings made here might vanish again, depending on the TTY driver
          * used. A 2nd ("constructive") initialization after we opened the input/output fds we actually want
-         * will fix this. */
-        exec_context_tty_reset(context, params);
+         * will fix this. (Note that we pass a NULL context ID here, this expects the context ID to close,
```

Comment:
`this expects` -> `which expects` ?
Anyway, the comment is slightly hard to understand for me.
I guess, as OSC sequence is not opened yet, it is not necessary and should not close OSC sequence, hence invocation ID should not be passed here. Right?